bindings/py/cpp_src/bindings/encoders/py_CoordinateEncoder.cpp

```diff
@@ -0,0 +1,70 @@
+/* ----------------------------------------------------------------------
+ * Numenta Platform for Intelligent Computing (NuPIC)
+ * Copyright (C) 2019, David McDougall
+ * The following terms and conditions apply:
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses.
+ *
+ * http://numenta.org/licenses/
+ * ---------------------------------------------------------------------- */
+
+#include <bindings/suppress_register.hpp>  //include before pybind11.h
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <nupic/encoders/CoordinateEncoder.hpp>
+
+namespace py = pybind11;
+
+using namespace std;
+using namespace nupic;
+using namespace nupic::encoders;
+using nupic::sdr::SDR;
+
+namespace nupic_ext
+{
```

Comment:
put the import `using ...` within this ns.

Reply:
This is a source file, should never be imported by anyone else...